yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -346,7 +346,13 @@ class SkipOperator : public FlowableOperator<T, T> {
      void request(int64_t delta) override {
        if (firstRequest_) {
          firstRequest_ = false;
-         delta = delta + offset_;
+
+         // TODO: This should be helper like `clampAdd` or something.
+         if (delta <= std::numeric_limits<int64_t>::max() - offset_) {
```

Comment:
Consider using `SubscriptionHelper` https://github.com/rsocket/rsocket-cpp/blob/master/yarpl/src/yarpl/flowable/utils/SubscriptionHelper.h#L39

It handles the limits, overflow, etc in one standardized and tested place.